flink-runtime/src/test/java/org/apache/flink/runtime/rest/handler/legacy/JobVertexBackPressureHandlerTest.java

```diff
@@ -57,7 +57,7 @@ public void testGetPaths() {
 	@Test
 	public void testResponseNoStatsAvailable() throws Exception {
 		ExecutionJobVertex jobVertex = mock(ExecutionJobVertex.class);
-		BackPressureStatsTracker statsTracker = mock(BackPressureStatsTracker.class);
+		BackPressureStatsTrackerImpl statsTracker = mock(BackPressureStatsTrackerImpl.class);
```

Comment:
we could replace the mocks in this class with VoidBackPressureStatsTracker / lambda implementations.

Reply:
Will change it.

Reply:
No unfortunately not, because the `JobVertexBackPressureHandler` requires `BackPresureStatsTrackerImpl` because of some methods which are not part of the interface.